spec/spec_helper.rb

```diff
@@ -72,6 +72,7 @@ def coverage_needed?
   browser_options.args << '--headless'
   browser_options.args << '--disable-gpu'
   browser_options.args << '--no-sandbox'
+  browser_options.args << '--window-size=1920, 1080'
```

Comment:
Can you please explain why this change is being added?  Was it needed for tests to pass? Or could it be removed?

Reply:
This was required to pass, I can rebase and attempt to remove this